server/spec/consumer_resource_spec.rb

```diff
@@ -91,15 +108,33 @@
         nil,
         0,
         nil,
-        'offer',
-        'u-sage')
-      consumer['offering'].should == 'offer'
+        'role',
+        'u-sage',
+        ['add-on 1', 'add-on 2'])
+      consumer['role'].should == 'role'
       consumer['usage'].should == 'u-sage'
 
-      @user2.update_consumer({:uuid => consumer['uuid'], :offering => 'updatedoffering', :usage => 'updatedusage'})
+      add_ons = consumer['addOns']
+      add_ons.size.should == 2
+      add_ons.should include('add-on 1')
+      add_ons.should include('add-on 2')
+
+      update_args = {
+          :uuid => consumer['uuid'],
+          :role => 'updatedrole',
+          :usage => 'updatedusage',
+          :addOns => ['add-on 2', 'add-on 4']
```

Comment:
Should we add SLA to this test too since that's part of the system purpose matrix?

Reply:
Done. Will be included in next update of this PR. Thanks!